src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -943,6 +993,124 @@ public HttpClientOptions setLogActivity(boolean logEnabled) {
   }
 
   /**
+   * Set option to offer Deflate Frame websocket compression
+   * @param tryDeflateFrame
+   * @return  a reference to this, so the API can be used fluently
+   */
+ public HttpClientOptions setTryWebsocketDeflateFrameCompression(boolean tryDeflateFrame)
+ {
+	 this.websocketTryUseDeflateFrame = tryDeflateFrame;
+	 return this;
+ }
+ 
+ /** 
+  * 
+  * @return Whether Deflate Frame websocket compression will be offered
+  */
+ public boolean isTryWebsocketDefalteFrameCompressionEnabled()
+ {
+	 return this.websocketTryUseDeflateFrame;
+ }
+ 
+/**
+ * Set option to offer Permessage Deflate websocket compression
+ * @param tryPermessageDeflate
+ * @return  a reference to this, so the API can be used fluently
+ */
+ public HttpClientOptions setTryWebsocketTryPermessageDefalteCompression(boolean tryPermessageDeflate)
```

Comment:
Are you sure? This is the function to set permessage deflate. Also, in the previous comment for the deflate frame compression setter, you recommended removing the 'set' prefix from the name.